pkgs/top-level/coq-packages.nix

```diff
@@ -41,11 +41,11 @@ let
       coq-haskell = callPackage ../development/coq-modules/coq-haskell { };
       coq-lsp = callPackage ../development/coq-modules/coq-lsp {};
       coq-record-update = callPackage ../development/coq-modules/coq-record-update { };
-      coqeal = callPackage ../development/coq-modules/coqeal (
+      coqeal = callPackage ../development/coq-modules/coqeal 
         (lib.optionalAttrs (lib.versions.range "8.13" "8.14" self.coq.coq-version) {
           bignums = self.bignums.override { version = "${self.coq.coq-version}.0"; };
         })
-      );
+      ;
```

Comment:
And here

Reply:
This should be ignored then.  "double" parenthesized !